### DISCUSSION TITLE
알고리즘 풀이 언어 관련 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#97] [Q&A] [answered]
최근에 꾸준하게 스케줄링하여 시도해보고 싶은 중 하나가 알고리즘 풀이입니다.

취업 목적이 아닌 알고리즘 문제 자체에 대한 이해정도를 하고 싶어서 다양한 장르를 풀어보고 싶은데 과거에는 C++로 풀었다가 이번에 다시 풀려고 하니 고민이 되어 질문합니다.

첫 번째는 가장 많이 사용한 언어, 편한 언어가 C#인데 대부분 인터넷의 내용이나 주위에서 파이썬, c++로 알고리즘을 풀이합니다.

당장 취업목적도 아니고 언어에 대한 스킬이나 문제해결 능력을 기르고 싶은 거라면 가능하다면 C#으로 풀어도 문제가 없을까요?

다른 시선으로 한 가지 언어에 매몰되지 않고 다양한 언어도 써보기에 좋은 기회일까요?

아니면 한 문제에 대해 두 가지 언어를 써보는 것이 좋을까요..?

이런 질문은 사실 제 선택이 가장 중요하지만 멘토님의 생각이 궁금해서 구구절절 적어봤습니다..

**Comment by jongfeel** [comment, marked as answer]
우선 결정하신 대로 하는게 제일 좋을 것 같고요.

제 의견을 드리자면 언어는 중요하지 않습니다.
다양한 언어를 쓰는 기회라고 보고 싶다면 그게 좋을 것 같긴 합니다.
